packages/wallet/ui/src/components/ImportContact.jsx

```diff
@@ -84,7 +84,7 @@ export const ImportContactWithoutContext = ({
             variant="standard"
             InputProps={{
               startAdornment: (
-                <InputAdornment position="start">board:</InputAdornment>
+                <InputAdornment position="start">board0</InputAdornment>
```

Comment:
We might want to move this below the input rather than in the box, like "board0..." as a help tip. When it's at the start, it's unclear that you should still include "board0" in the input

Reply:
> We might want to move this below the input rather than in the box, like "board0..." as a help tip.

I'd gladly approve this kind of change next time you're working nearby.

>  When it's at the start, it's unclear that you should still include "board0" in the input

Agreed.